jax/interpreters/xla.py

```diff
@@ -279,7 +279,8 @@ def check_nans(prim, bufs):
 
 def _check_nans(name, xla_shape, buf):
   assert not xla_shape.is_tuple()
-  if dtypes.issubdtype(xla_shape.element_type(), onp.floating):
+  if (dtypes.issubdtype(xla_shape.element_type(), onp.floating) or
```

Comment:
`floating` and `complexfloating` have a common supertype, namely `inexact`. Use it instead?

https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.scalars.html